src/components/ProductFilterModal.vue

```diff
@@ -84,26 +92,47 @@ export default defineComponent({
       modalController.dismiss({ dismissed: true, isFilterChanged: this.isFilterChanged });
     },
     async search(event: any) {
-      // TODO: need to implement infinite scroll on the modal search
+      this.queryString = event.target.value;
+      this.facetOptions = [];
```

Comment:
I think we should handle it in get tags only

Reply:
Yes sir.